hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/AbstractTraceEnricher.java

```diff
@@ -69,10 +70,14 @@ protected void addEnrichedAttribute(Event event, String key, AttributeValue valu
     enrichedAttributes.getAttributeMap().put(key, value);
   }
 
-  protected void addEnrichedAttributes(Event event, List<Pair<String, AttributeValue>> attributes) {
+  protected void addEnrichedAttributes(
+      Event event,
+      List<Pair<String, AttributeValue>> attributes,
```

Comment:
Can we just pass in a map here instead of List & Map?

Reply:
Yup. Planning to refactor that as well. probably in a separate PR.